outputs.tf

```diff
@@ -15,59 +15,54 @@
  */
 
 output "project_id" {
-  value = "${local.project_id}"
+  value = "${module.project-factory.project_id}"
 }
 
 output "project_number" {
-  value = "${local.project_number}"
+  value = "${module.project-factory.project_number}"
 }
 
 output "domain" {
-  value       = "${local.domain}"
+  value       = "${module.project-factory.domain}"
   description = "The organization's domain"
 }
 
-output "group_email" {
```

Comment:
We should maintain this as a pass-through output (just reflecting back the input, if any).